projects/kit/components/input-tag/input-tag.component.ts

```diff
@@ -140,6 +140,9 @@ export class TuiInputTagComponent
     @Input()
     uniqueTags = this.options.uniqueTags;
 
+    @Input()
+    tagAutoColor = this.options.tagAutoColor;
```

Comment:
Let's rename `tagAutoColor` to `autoColor`. There's nothing to "color" here other than tags